.gitmodules

```diff
@@ -106,3 +106,6 @@
 [submodule "dependencies/3rdparty/draco"]
 	path = dependencies/3rdparty/draco
 	url = git@github.com:leggedrobotics/draco.git
+[submodule "dependencies/internal/transfolder"]
```

Comment:
using a new message type from transfolder to compare the restored submaps with all available submaps from transfolder, therefore adding this as a submodule here